exercises/practice/ocr-numbers/t/ocr-numbers.rakutest

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env raku
+use Test;
+use lib $?FILE.IO.parent(2).add('lib');
+use OcrNumbers;
+
+cmp-ok( # begin: 5ee54e1a-b554-4bf3-a056-9a7976c3f7e8
+    ascii-to-digits(" _ \n| |\n|_|\n   "),
```

Comment:
I wonder if using Heredocs would make this more readable to the reader of the tests.

Reply:
If the terminator is indented, that amount of indention is removed from the string literals.  So I am unsure of the indentation changes being a problem.

However, you are closer to this situation than I am.